Marlin/src/feature/power.cpp

```diff
@@ -60,6 +60,10 @@ bool Power::psu_on;
   millis_t Power::lastPowerOn;
 #endif
 
+#if defined(PS_ON_EDM_PIN) || (defined(PS_ON_EDM_PIN) && ENABLED(PSU_OFF_REDUNDANT))
```

Comment:
The second one should be `PS_ON1_EDM_PIN`. But I presume that you must have `PS_ON_EDM_PIN` before you're allowed to have `PS_ON1_EDM_PIN` so the second condition can be removed.

Reply:
First note is correct, however its more the opposite on the second. If you only have a single EDM on a redundant output, you would follow the second relay as the "business end" of the output control, usually using the actual safe power output as the return. I suppose you could name it to follow the first output, however for clarity sake probably best to keep it following the actual device its linked to.